### PR TITLE
Update docs to highlight normalization support in configuration cache

### PR DESCRIPTION
**Pull request by mlopatkin**
Blocked on #18293

### CODE REVIEW

**Review by eskatos — APPROVED** [review]
👍

File: subprojects/docs/src/docs/release/notes.md [hunk 1]
```diff
@@ -108,6 +108,10 @@ However, these events can be unreliable in some environments, which could cause
 To prevent this, Gradle now makes a change to the file system whenever it first starts watching something.
 If a file system event for this change does not arrive by the start of the next build, Gradle concludes that file system events are unreliable, and will fall back to checking the file system for files involved in the build for changes.  
 
+### Input normalization support in configuration cache
+The [input normalization](userguide/more_about_tasks.html#sec:configure_input_normalization) is now correctly tracked by the configuration cache. Task up-to-date checks now consider normalization
```
Comment (eskatos) [inline subprojects/docs/src/docs/release/notes.md:112]:
```suggestion
The [input normalization](userguide/more_about_tasks.html#sec:configure_input_normalization) is now correctly tracked by the experimental [configuration cache](userguide/configuration_cache.html). Task up-to-date checks now consider normalization
```

**Review by eskatos — CHANGES_REQUESTED** [review]
I added a suggestion to state that the configuration cache is still experimental and add a link to the dedicated manual chapter for readers that are not aware of it yet.

**Comment by mlopatkin** [comment]
> I added a suggestion to state that the configuration cache is still experimental and add a link to the dedicated manual chapter for readers that are not aware of it yet.

Thanks, updated

**Comment by mlopatkin** [comment]
@bot-gradle test and merge

**Review by eskatos — APPROVED** [review]

**Pull request merged**